pages/api/discover/random.js

```diff
@@ -1,7 +1,8 @@
 import connectMongo from "@config/mongo";
 import logger from "@config/logger";
-import Profile from "@models/Profile";
+import loadProfiles from "@services/profiles/loadProfiles";
```

Comment:
Not sure why this import was added? It is not used in the file, maybe from a bad merge (or git conflict)